src/MarginTrading.Backend.Core/Settings/MarginSettings.cs

```diff
@@ -33,5 +33,6 @@ public class MarginSettings
         [Optional]
         public virtual TelemetrySettings Telemetry { get; set; }
         public int MaxMarketMakerLimitOrderAge { get; set; }
+        public string EquivalentAssetSettings { get; set; }
```

Comment:
I suggest renaming this property to something more specific, ex "ReportingEquivalentPricesAsset".
The word "Settings" in the current variant alludes to an idea that there is something complex stored in there, instead of the simple asset name in reality.

Reply:
Fixed